### DISCUSSION TITLE
usbipd: error: Failed to attach device with BUSID <busID>

### DISCUSSION THREAD

**Discussion by DanielePaz** [microsoft/WSL#8419] [General]
### Version

Windows 11 21H2 (build SO 22000.613 )

### WSL Version

- [X] WSL 2
- [ ] WSL 1

### Kernel Version

5.10.102.1

### Distro Version

Ubuntu 20.4

### Repro Steps

![20220421_090809](https://user-images.githubusercontent.com/41476266/164710503-90960efe-1a31-417f-9063-eef30e9df059.jpg)

### Expected Behavior

![20220421_090809](https://user-images.githubusercontent.com/41476266/164709871-0a94363f-38a3-47c6-abe5-450f4deb42d5.jpg)
![20220421_084340](https://user-images.githubusercontent.com/41476266/164710264-8179dea2-3087-401c-a49e-9f53ac75403b.jpg)

### Actual Behavior

PS C:\Windows\System32> wsl --update
Ricerca aggiornamenti in corso...
Nessun aggiornamento disponibile.
Versione del kernel: 5.10.102.1
PS C:\Windows\System32> wsl --shutdown
PS C:\Windows\System32> usbipd wsl list
BUSID  DEVICE                                                        STATE
1-2    Realtek Bluetooth Adapter                                     Not attached
2-2    NexiGo N950P 4K Webcam, NexiGo N950P 4K Webcam Audio          Not attached
2-3    Dispositivo di input USB                                      Not attached
3-1    Integrated Camera                                             Not attached
3-3    ELAN WBF Fingerprint Sensor                                   Not attached

PS C:\Windows\System32> usbipd wsl attach --busid 2-2
usbipd: info: Using default distribution 'Ubuntu'.
usbip: error: Attach Request for 2-2 failed - Device busy (exported)

usbipd: error: Failed to attach device with BUSID '2-2'.

**Comment by DanielePaz** [comment]
Hi, 
I have used WSL2 to attach/detach a USB webcam device to/from Ubuntu distribution,
The first time the usb webcam has been attached, and after detached. 
But the issue came the second time when I tried to attached as new the usb webcam to Ubuntu distribution
using the same commands i.e : usbipd wsl attach --busid <busid>
but this time the WSL has show this error message : 

usbip: error: Attach Request for 2-2 failed - Device busy (exported)

usbipd: error: Failed to attach device with BUSID '2-2'.

I don't understand why the webcam is defined as "busy " and I don't know how to solve this issue,
because now I can't attach the usb webcam to Ubuntu distribution and I can't use it.

Thank you very much, if you could help me to solve this question
Best wishes
Daniele

**Reply by ghost** [reply]
I'd reach out or start a discussion here: https://github.com/dorssel/usbipd-win/issues
